vars/RedHat.yml

```diff
@@ -80,12 +80,19 @@ postgresql_packages:
 
 # Extra packages
 etcd_package_repo: "https://github.com/etcd-io/etcd/releases/download/v{{ etcd_version }}/etcd-v{{ etcd_version }}-linux-amd64.tar.gz"
-vip_manager_package_repo: "https://github.com/cybertec-postgresql/vip-manager/releases/download/v{{ vip_manager_version }}/vip-manager_{{ vip_manager_version }}_Linux_x86_64.rpm"
+vip_manager_package_repo_parts:
```

Comment:
it seems to me that we are slightly complicating the management due to the desire to shorten the length of the string.

I would like to avoid creating additional variables like xx_parts 
Since variable files are what users interact with directly, we need to try to ensure convenience and readability. 

Personally, I don't think a long string in a variable is a bad thing. Yes, it may make sense for code, but it doesn't always make sense for variables. 

This is just my opinion and I may be wrong because I am not a developer, I am a DBA.

Reply:
I will make the necessary corrections to avoid having any "xx_parts".

Please refrain from suggesting that we should disregard the yaml[line-length] rule 🤣

Reply:
we can disable the rule at the level of a single line or the entire variable file. Right?

Reply:
@vitabaks

As a DBA and sysAdmin until 2019, my skills in Postgres are beginning to rust. Fortunately, you are here 😇 

Allow me to provide further details about my professional background. You can refer to my LinkedIn profile at the following link: [Thomas Sanson](https://www.linkedin.com/in/sansonthomas/)

It is indeed possible to ignore the rule on a file or line basis using the comment `# yamllint disable rule:line-length`. However, I would prefer to find a solution that satisfies everyone

I will explore how to address this matter, and if you notice any other aspects that trouble you, please do not hesitate to share. I appreciate the way we engage in constructive discussions